test/python/transpiler/test_optimize_1q_decomposition.py

```diff
@@ -684,7 +684,7 @@ def test_if_else(self):
         test = QuantumCircuit(qr, cr)
         test.h(0)
         test.measure(0, 0)
-        test_true = QuantumCircuit(qr)
+        test_true = QuantumCircuit(qr, [cr[0]])
```

Comment:
Is this different to the following?
```suggestion
        test_true = QuantumCircuit(qr, cr)
```

Reply:
No, should be the exact same.  I'll fix the other instance in this file as well.